### DISCUSSION TITLE
[Proposed] Print kind: "Digital" - Print

### DISCUSSION THREAD

**Discussion by AmirSimantov** [PanditProject/Website#13] [Help] [answered]
@OdySatriyo (a data inspector) has stumbled upon a Print entity that is really a website. It first looked to me out of Pandit's scope. However, @chakrabortydeepro explains that the Print entity represents a manuscript catalog with only an online representation and wants to reference using the "Scholarship" field of a manuscript. it makes sense. (The entity is [Bharatiya Kriti Sampada](https://www.panditproject.org/entity/109736/print)).

Although Pandit's scope isn't about librarian semantics, this needs attention.

We do have an option "Unpublished," and I can see that you, contributors, use it for handlists and unpublished articles ([see filtered results](https://www.panditproject.org/search/print/k/unpubished)). Have any of these sources been digitally born?

It sounds weird to call a website or an online-only source "Unpublished" as if there is ever an intention to "publish" it. By the way, PanditProject.org is such a source, as well.

I tend to add an option called "Digital" in the field "Print kind" (albeit weird because 'digital' and 'print' sound opposites; we may end up one day renaming 'Print'). If we do, we may change some of the Print entities that now have the value "Unpublished" to be "Digital." (There is also an entity [Gupta 2020](https://www.panditproject.org/entity/108630/print) that we may change to "Digital.")

What's your take on that?

Thanks.

**Comment by mark-rozen** [comment, marked as answer]
I would suggest using the term "Web page" since in citation guides of many fields there is a specific style for this kind of source. "Digital" is a too wide term, it could be many things. Moreover, online books and articles are still books and articles, so I don't think we need to change their type to "Digital". I know that PanditProject is not as much as a librarian tool, but I think scholars could also use it for citation purposes, and defining this as a "Web page" will clarify a lot. 
Of course, I might be wrong :)

**Reply by AmirSimantov** [reply]
Thanks, @mark-rozen. Can we say that, in contrast to an "Unpublished" Print entity that can have copies in multiple URLs, a "Web page" Print entity can only be in a single URL? 
If yes, what about Sanskrit MSS Cat. Srinagar ([entity/109103/print](https://www.panditproject.org/entity/109103/print))? I can see that one of the links directs to a "real" _website_ page while the others to copies(?) of the content(?) in that website?

**Reply by mark-rozen** [reply]
As far as I see, it's more a collection of different PDFs (so, unpublished a good term for it) rather than a regular Website. I confused a little bit and don't know what is the better definition in that case. In my understanding, yes, a Website entity can have only one link.

**Comment by wujastyk** [comment]
I agree with Mark.

Regarding the taxonomy of bibliographical resources, I suggest following
someone else's scheme.  No need to reinvent the wheel.  Something like
Zotero or BibLaTeX, or perhaps there's an ISO or LC typology out there.  (ISO
690
<https://en.wikipedia.org/wiki/ISO_690#Examples_of_citation_(ISO_690:1987)>?
BibTeX <https://en.wikipedia.org/wiki/BibTeX#Entry_types>, EndNote
<https://en.wikipedia.org/wiki/EndNote#Tags_and_fields>, RIS
<https://en.wikipedia.org/wiki/RIS_(file_format)#Type_of_reference>, Zotero
<https://www.zotero.org/support/kb/item_types_and_fields>, etc.)  Or one of
the standard bibliography guides like Chicago Manual of Style.
Best,
Dominik

**Reply by AmirSimantov** [reply]
Thank you, @wujastyk, you are correct. We may do it in the future if we will have the resources also to revise all the data that was entered. This discussion was just the iceberg.

**Comment by AmirSimantov** [comment]
I have added a "Web page" kind.

@mark-rozen - could you please have a look at the [Print entities with "Unpublished" kind](https://www.panditproject.org/search/print/k/unpubished)? Please let me know which of them should be changed to "Web page" (only an administrator can change the Print kind, you currently can't).

**Reply by mark-rozen** [reply]
In my opinion, all of them should stay "Unpublished" as they are since none of them is really fit the type of "Web page".

**Reply by AmirSimantov** [reply]
Thank you @mark-rozen!
